core/wiki/macros/CSS.tid

```diff
@@ -74,3 +74,47 @@ column-count: $columns$;
 \define if-background-attachment(text)
 <$reveal state="$:/themes/tiddlywiki/vanilla/settings/backgroundimage" type="nomatch" text="">$text$</$reveal>
 \end
+
+\define if-sticky-titles-calc-zindex()
+``
+[data-tiddler-title="``$(cssEscapedTitle)$``"] .tc-tiddler-title {
+	z-index: calc(500 + ``$(storyCount)$`` - ``$(tiddlerIndex)$``);
```

Comment:
Perhaps we should have a fallback declaration for browsers that don't have "calc"?

Reply:
Yes probably, I've tried finding some good fallback options but I haven't found any. Can you suggest one?

Alternativelly, I have a fully working version without calc, just `z-index: $(value)$`